openmetadata-spec/src/main/resources/json/schema/tests/testSuite.json

```diff
@@ -151,6 +151,11 @@
         "$ref": "../type/tagLabel.json"
       },
       "default": null
+    },
+    "inherited": {
```

Comment:
I dont see this used anywhere. Is it left here by mistake?

Reply:
it's used in the search client when propagating fields from parent to childs